modules/persistence/build.gradle.kts

```diff
@@ -0,0 +1,51 @@
+plugins {
+    kotlin("multiplatform") version "2.0.0"
+    id("app.cash.sqldelight") version "2.0.2"
+}
+
+group = "com.sphereon.oid.fed.persistence"
+version = "0.1.0"
+
+repositories {
+    google()
+    mavenCentral()
+    mavenLocal()
+}
+
+sqldelight {
+    databases {
+        create("Database") {
+            dialect("app.cash.sqldelight:postgresql-dialect:2.0.2")
+            packageName.set("com.sphereon.oid.fed.persistence")
+            srcDirs.from(project.file("./src/commonMain/sqldelight/"))
+        }
+    }
+}
+
+kotlin {
+    jvm{
+        withJava()
+    }
+
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation(projects.modules.openapi)
+            }
+        }
+
+        jvmMain {
+            dependencies {
+                implementation("com.zaxxer:HikariCP:5.1.0")
+                implementation("org.postgresql:postgresql:42.7.3")
+                implementation("app.cash.sqldelight:jdbc-driver:2.0.2")
+            }
+        }
+    }
+}
+
+java {
+    toolchain {
+        languageVersion = JavaLanguageVersion.of(21)

```

Comment:
Will not work for multiplatform. It should be done in the jvm target. Ex:
`kotlin {
    jvm {
        tasks {
            named<KotlinJvmCompile>("compileKotlinJvm") {
                dependsOn("fixOpenApiGeneratorIssue")
                compilerOptions {
                    jvmTarget.set(JvmTarget.JVM_11)
                }
            }
        }
    }
}`

Reply:
would this suffice?
`    @OptIn(ExperimentalKotlinGradlePluginApi::class)
    compilerOptions {
        apiVersion.set(org.jetbrains.kotlin.gradle.dsl.KotlinVersion.KOTLIN_2_1)
    }
`